### PR TITLE
 Add support for Objective-C super calls

### PR DESCRIPTION
**Pull request by jacob-carlborg**

### CODE REVIEW

**Comment by jacob-carlborg** [comment]
@adamdruppe Super calls implemented 😃.

**Review by thewilsonator — APPROVED** [review]

**Comment by thewilsonator** [comment]
Please update the headers.

**Comment by jacob-carlborg** [comment]
> Please update the headers.

@thewilsonator which headers?

**Comment by thewilsonator** [comment]
Sorry I thought this changed `objc.d` not `objc_glue.d`, my bad.

**Comment by ibuclaw** [comment]
Update the headers anyway. :-D

File: src/dmd/objc_glue.d [hunk 1]
```diff
@@ -54,6 +54,12 @@ ObjcGlue objc()
 // Should be an interface
 extern(C++) abstract class ObjcGlue
```
Comment (ibuclaw) [inline src/dmd/objc_glue.d]: This doesn't require C++ linkage anymore does it?

Any backend support in gdc/ldc won't be using this module IIUC.

Comment (jacob-carlborg) [reply src/dmd/objc_glue.d]: > This doesn't require C++ linkage anymore does it?

No.

**Review by WalterBright — APPROVED** [review]

**Comment by jacob-carlborg** [comment]
Three days are up.

**Comment by jacob-carlborg** [comment]
Thanks.

**Pull request merged**